product_docs/docs/pgd/5/quickstart/index.mdx

```diff
@@ -21,7 +21,10 @@ EDB Postgres Distributed (PGD) is a multi-master replicating implementation of P
 
 !!! Note Fully managed BigAnimal
     If you prefer to have a fully managed EDB Postgres Distributed experience, PGD is now available as an option on BigAnimal, EDB's cloud platform for Postgres. Read more about [BigAnimal distributed high-availability clusters](/biganimal/latest/overview/02_high_availability/distributed_highavailability/).
- 
+
+!!! Note PGD Operator For Kubernetes
```

Comment:
@djw-m Going back and forth on these two blue notes and wanted to get your thoughts

Part of me wonders if instead of having BigAnimal and Kubernetes called out in the blue informational boxes, if we should just have them mentioned in a sentence? 

i.e. "This quick start guide walks through using PGD in on-premises and on VMs in self-managed environments. If you prefer to have a fully managed EDB Postgres Distributed experience, PGD is now available as an option on BigAnimal. Or, if you prefer to deploy PGD on Kubernetes, you can use the EDB PGD Operator for Kubernetes."

Thinking is how do we inform the users without necessarily drawing too much attention and distracting from the main message of the page? Know the impetus for this was my ticket about including breadcrumbs but thinking how best we can incorporate - Open to thoughts and what best practice is

Reply:
Shrunk the boxes into one and bulleted them.